pymysql/util.py

```diff
@@ -17,3 +17,18 @@ def join_bytes(bs):
         for b in bs[1:]:
             rv += b
         return rv
+
+# https://dev.mysql.com/doc/internals/en/integer.html#packet-Protocol::LengthEncodedInteger
+def lenenc_int(i):
```

Comment:
All of these are protocol related.
And all code relating to protocol is in `connections` module for now.